src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -89,4 +90,16 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses {@code Collection<String> decks} into a {@code Set<Deck>}
+     */
+    public static Deck parseDeck(String deckName) throws ParseException {
+        requireNonNull(deckName);
+        String trimmedDeckName = deckName.trim();
+        if (!Answer.isValidAnswer(trimmedDeckName)) {
```

Comment:
Maybe in the future we can use another method name for this? Something like `isValidDeckName' or something to avoid confusion.